partner_create_by_vat/res_partner.py

```diff
@@ -191,40 +75,76 @@ def button_get_partner_data(self):
             self.write({'vat_subjected': False})
 
         if vat_number and vat_country:
-            if vat_number and vat_country and vat_country.upper()=='RO':
-                res = requests.get('http://openapi.ro/api/companies/' + str(vat_number) + '.json')
-                if res.status_code==200:
-                    res = json.loads(res.content)
-                    state = False
-                    if res['state']:
-                        state = self.env['res.country.state'].search([('name','=',res['state'].encode('utf-8').title())])
-                        if state:
-                            state = state[0].id
+            if vat_country.upper()=='RO':
+                self.write({
+                    'is_company': True,
+                    'country_id': self.env['res.country'].search([('code','=','RO')])[0].id
+                })
+                try:
+                    result = getMfinante(vat_number)
+                    name = nrc = adresa = tel = fax = zip1 = vat_s = state = False
+                    if 'Denumire platitor:' in result.keys():
+                        name = result['Denumire platitor:'].upper()
+                    if 'Adresa:' in result.keys():
+                        adresa = result['Adresa:'].title() or ''
+                    if 'Numar de inmatriculare la RegistrulComertului:' in result.keys():
+                        nrc = result['Numar de inmatriculare la Registrul Comertului:']
+                        if nrc == '-/-/-':
+                            nrc = ''
+                        else:
+                            nrc = nrc.replace(' ', '')
+                    if 'Codul postal:' in result.keys():
+                        zip1 = result['Codul postal:'] or ''
+                    if 'Judetul:' in result.keys():
+                        state = False
+                        jud = result['Judetul:'].title() or ''
+                        if jud:
+                            state = self.env['res.country.state'].search([('name','ilike',jud)])
```

Comment:
Aici ar trebui sa verificam..la mfinante apare Municipiul Bucuresti care in res.country.states e numai Bucuresti...sa facem replace pe "Municipiul " ca sa il recunoasca...